server/core/terraform/terraform_client.go

```diff
@@ -394,9 +395,15 @@ type Line struct {
 // Callers can use the input channel to pass stdin input to the command.
 // If any error is passed on the out channel, there will be no
 // further output (so callers are free to exit).
-func (c *DefaultClient) RunCommandAsync(ctx models.ProjectCommandContext, path string, args []string, customEnvVars map[string]string, v *version.Version, workspace string) (chan<- string, <-chan Line) {
+func (c *DefaultClient) RunCommandAsync(ctx models.ProjectCommandContext, path string, args []string, customEnvVars map[string]string, v *version.Version, workspace string) <-chan Line {
+
+	input := make(chan string)
```

Comment:
Seems like this introduces a mixed convention for naming channels inCH/outCH vs input. We might want to follow the convention if we're upstreaming or change the outCh to just output.

Reply:
Actually looks like this is the only channel defined in this function which seems fine.

Reply:
Yeah I can make it consistent.